public/app/features/explore/state/main.ts

```diff
@@ -108,9 +108,9 @@ export const splitOpen: SplitOpen = (options): ThunkResult<void> => {
     let rightUrlState: ExploreUrlState = leftUrlState;
 
     if (options) {
-      const datasourceName = getDataSourceSrv().getInstanceSettings(options.datasourceUid)?.name || '';
+      const datasourceUid = getDataSourceSrv().getInstanceSettings(options.datasourceUid)?.uid || '';
```

Comment:
**Suggestion:** I think we can just use `options.datasourceUid` straight away?

Reply:
🤦‍♀️ thank you!